ArduCopter/ArduCopter.cpp

```diff
@@ -485,7 +485,10 @@ void Copter::one_hz_loop()
 
         update_using_interlock();
 
-#if FRAME_CONFIG != HELI_FRAME
+#if FRAME_CONFIG == HELI_FRAME
```

Comment:
I think IF/ELSEs like this should be separated into two different IFs since the two outcomes are unrelated from a functional perspective.